src/vgr_cairo.mli

```diff
@@ -0,0 +1,117 @@
+(*---------------------------------------------------------------------------
+   Copyright 2014 Arthur Wendling, Daniel C. Bünzli. All rights reserved.
+   Distributed under the BSD3 license, see license at the end of the file.
+   %%NAME%% release %%VERSION%%
+  ---------------------------------------------------------------------------*)
+
+(* Based on the Vgr_htmlc documentation by Daniel C. Bünzli. *)
+
+(** Vg Cairo renderer.
+
+    {b Dependencies:}
+    {ul {- {e {{:http://forge.ocamlcore.org/projects/cairo/}Cairo2 bindings
+     for OCaml}}}
+     {- {e {{:http://cairographics.org/}Cairo Graphics library}}}}
+
+    {e Release %%VERSION%% — %%MAINTAINER%% } *)
+
+(** {1:target Cairo render targets} *)
+
+val target : ?resolution:Gg.V2.t -> [< `PDF | `PNG | `PS | `SVG ] ->
+  Vg.Vgr.dst_stored Vg.Vgr.target
+(** [target resolution fmt] is a render target for rendering to the stored
+    destination given to {!Vg.Vgr.create} in the chosen format [fmt].
+    {ul
+    {- [resolution], specifies the rendering resolution in samples per
+       meters. The PDF, PS and SVG formats are measured in points by Cairo,
+       while the PNG format is in pixels. If unspecified, the default
+       conversion to points is used.}}
```

Comment:
This comment is hard to understand. Is resolution that [resolution] argument expressed in samples per meters or not ? (P.S. do not clarify yourself I'm currently merging in another branch).